ci-operator/config/openshift/library-go/openshift-library-go-master.yaml

```diff
@@ -29,23 +29,25 @@ tests:
     make verify -k --warn-undefined-variables
   container:
     from: bin
+- as: verify-deps
+  commands: |
+    make verify-deps -k --warn-undefined-variables
+  container:
+    from: bin
 - as: verify-alpha-build-machinery
   commands: |
-    HOME=$( mktemp -d )
     make -C alpha-build-machinery/ verify -k --warn-undefined-variables
   container:
     from: src
 - artifact_dir: /tmp/artifacts
   as: unit
   commands: |
-    mkdir -p /tmp/artifacts
```

Comment:
this was a temporary hack for broken ci-operator that stopped creating the dir at some point in the past